pkg/api/v1/defaults.go

```diff
@@ -92,6 +92,15 @@ func addDefaultingFuncs() {
 			// If limits are specified, but requests are not, default requests to limits
 			// This is done here rather than a more specific defaulting pass on ResourceRequirements
 			// because we only want this defaulting semantic to take place on a Pod and not a PodTemplate
+			_, found := obj.Annotations["kubernetes.io/config.mirror"]
```

Comment:
Actually, admission control probably isn't a problem, but it would still be useful to not generate a diff with 1.1 kubelets.